src/ssz/type/container.zig

```diff
@@ -156,6 +156,20 @@ pub fn FixedContainerType(comptime ST: type) type {
             }
         };
 
+        pub fn serializeIntoJson(allocator: std.mem.Allocator, writer: anytype, in: *const Type) !void {
+            try writer.beginObject();
+            inline for (fields) |field| {
+                const field_value = @field(in, field.name);
```

Comment:
in typescript we support different type of cases: eth2, snake, constant, camel, header, pascal. Would be nice if our zig implementation could do the same thing, not sure its effort to implement and the use case of it